.gitignore

```diff
@@ -6,7 +6,6 @@
 *.dylib
 bin
 testbin/*
-api/*/zz*generated.deepcopy.go
```

Comment:
Why?

Reply:
The reason I am removing this from .gitignore is that this generated.deepcopy.go file is needed in eks-a.
eks-a code is referring capc package, and some code (client.object) requires this generated.deepcopy.go code.

CAPC's previous release has this file, after removing this file, eks-a could not build.

Also, other package like vsphere has this generated deepcopy file exposed.

Reply:
I don't want to hang up our release, but after the release I would like us to review this again.